multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -27,6 +27,8 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
At mc-controller restarts, we should remove stale clusters in the ClusterSet, whose MemberAnnounces are already deleted before controller starts.